src/parsers/js/acorn-loose.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'acorn/package.json';
+import jsxPkg from 'acorn-jsx/package.json';
+import * as LocalStorage from '../../LocalStorage';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'acorn-loose';
+const options = {
+  ecmaVersion: 6,
+  ranges: true,
+  sourceType: 'module',
+  ...LocalStorage.getParserSettings(ID),
+};
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: `${pkg.version}`,
+  homepage: pkg.homepage,
+  locationProps: new Set(['range', 'loc', 'start', 'end']),
+
+  loadParser(callback) {
+    let _localReq = require;
+    require(['acorn'], (acorn) => {
+      global.acorn = acorn;
+      _localReq(['acorn/dist/acorn_loose'], callback);
```

Comment:
The `acorn/dist/acorn_loose.js` file is built with Browserify, which means that the Webpack `require` function cannot get in there. Hence these hacks.